example/src/androidTestDebug/java/com/stripe/example/activity/AddNetbankingPaymentMethodTest.kt

```diff
@@ -66,7 +66,7 @@ class AddNetbankingPaymentMethodTest {
     private fun launchBankSelector() {
         // launch Netbanking selection activity
         onView(withId(R.id.examples)).perform(
-            RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(13, click())
+            RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(11, click())
```

Comment:
This also needed to be updated because I removed an item from the list above this. But this previously was actually testing Fpx, not Netbanking :/ (basically the old index was wrong, which is why this is updated by two instead of one)